probe/cri/registry_test.go

```diff
@@ -0,0 +1,27 @@
+package cri_test
+
+import (
+	"testing"
+
+	"github.com/weaveworks/scope/probe/cri"
+)
+
+func TestParseHttpEndpointUrl(t *testing.T) {
+	_, err := cri.NewCRIClient("http://xyz.com")
+
+	if err == nil {
```

Comment:
This is fine, however you might want to look at `assert.Error` from the package `github.com/stretchr/testify/assert`.
We use this package in a few places, e.g. https://github.com/weaveworks/scope/blob/aaa63bbed0c91945477780b96d0be02c7cb9c3c6/prog/main_test.go#L22

It just neatens things up a bit compared to the three-line if-statement.

Reply:
Sure, that sounds good. I'll do it